axelor-stock/src/main/java/com/axelor/apps/stock/service/TrackingNumberConfiguratorService.java

```diff
@@ -0,0 +1,31 @@
+package com.axelor.apps.stock.service;
+
+import com.axelor.apps.base.db.Sequence;
+import com.axelor.apps.base.service.BarcodeGeneratorService;
+import com.axelor.apps.base.service.administration.SequenceService;
+import com.axelor.apps.base.service.app.AppBaseService;
+import com.axelor.apps.stock.db.TrackingNumberConfiguration;
+import com.axelor.exception.AxelorException;
+import com.axelor.inject.Beans;
+import com.google.inject.Inject;
+
+public class TrackingNumberConfiguratorService {
```

Comment:
Please create an interface for this service and rename this class TrackingNumberConfiguratorServiceImpl